### PR TITLE
Add "command color" property for font color

### PR DESCRIPTION
**Pull request by Crystalwarrior**
Give "comment" command a custom color
image

### CODE REVIEW

File: commands/command.gd [hunk 1]
```diff
@@ -112,6 +112,12 @@ var command_description:String :
 	set(value): return
 	get: return _get_description()
 
+## The color of this command's text
+## Used by the Comment command to fade the text out
+var command_color:Color :
```
Comment (AnidemDex) [inline commands/command.gd:117]: I think this need to be  `command_text_color`

File: commands/command.gd [hunk 2]
```diff
@@ -317,6 +323,9 @@ func _can_be_selected() -> bool:
 func _get_description() -> String:
 	return ""
 
+func _get_color() -> Color:
+	return Color()
```
Comment (AnidemDex) [inline commands/command.gd:327]: I think we should use `.get_theme_color("font_color", "Editor")`

Comment (Crystalwarrior) [reply commands/command.gd:327]: Collections are not a node but a resource so get_window().get_theme_color is unavailable

File: commands/command.gd (hunk 1 above)
Comment (Crystalwarrior) [reply commands/command.gd]: done

**Pull request merged**